.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```

Comment:
you'll want to add to your .gitignore file the `.idea` and `.idea/` . See the class gitignore here: https://github.com/UWPCE-PythonCert-ClassRepos/Wi2018-Online/blob/master/.gitignore